docsite-css/doc-site.css

```diff
@@ -27,6 +27,21 @@
 }
 /*******/
 
+/*** dark-mode ***/
+.dark-mode {
+  --background-color: #16191f;
+  --text-color: rgb(181, 192, 202);
```

Comment:
Since you're not using any alpha value in rgb value of var text-color, you can use HEX for better consistency.

Reply:
Good point I will make the changes accordingly. Thanks for the review.